README.md

```diff
@@ -71,6 +71,14 @@ Keep these goals in mind:
 You can use standard libraries to whatever extent is appropriate. In particular, you should not find it necessary to implement your own complete JSON parser.
 
 
+### Resources
```

Comment:
I'm inclined to keep the last sentence in there regardless. A little redundancy makes it that much easier to notice. I could be convinced otherwise, though.